### PR TITLE
add cc

### PR DESCRIPTION
**Pull request by shaharblanksela**
Add code coverage for pipeline + Report for PR

### CODE REVIEW

File: .github/workflows/ci-aws.yml [hunk 1]
```diff
@@ -135,6 +136,58 @@ jobs:
...
+          clover-file: packages/${{matrix.service}}/coverage/clover.xml
+      
+      - name: Find Comment Coverage Report
+        if: github.event_name == 'pull_request'
+        uses: peter-evans/find-comment@v2
+        id: fc
+        with:
+          issue-number: ${{ github.event.number }}
+          body-includes: "Total Coverage:"
+          direction: last
```
Comment (mshidlov) [inline .github/workflows/ci-aws.yml:158]: you can get a race condition on it (last comment) and get by mistake the wrong code cov -you can maybe output the cod coverage reports and write them to the pull request synchronically on a separated step.

Comment (shaharblanksela) [reply .github/workflows/ci-aws.yml]: There is no way to output code-coverage action. the workaround I've made is to create a new comment which created before the code coverage report synchronically on a different job.

**Comment by tupe12334** [comment]
@shaharblanksela What is that? can you close that if not in use?

**Comment by arielweinberger** [comment]
Closing as no response.

**Pull request closed without merging**